### PR TITLE
Fixed example DTCs after running on real CLI + added tutorial files

### PR DESCRIPTION
**Pull request by roy100**

### CODE REVIEW

File: docs/DTC Snippets.md [hunk 1]
```diff
@@ -29,12 +29,20 @@ Split: (time - end_time).minutes > 30
     Value: BlockAggregateName.id_to_value.set(source.id, source.value)
 ```
 
+<<<<<<< HEAD
```
[the remark this answers is not in the document]
Comment (roy100) [reply docs/DTC Snippets.md]: done

File: docs/DTC Snippets.md [hunk 2]
```diff
@@ -29,12 +29,20 @@ Split: (time - end_time).minutes > 30
...
+## Merge 2 lists in source data to create a list of tuples
+=======
 ## Merge 2 strings with comma separated values in source data to create a list of tuples
+>>>>>>> master
 
 ```yaml
 Fields:
   - Name: products_bought
     Type: list
+<<<<<<< HEAD
```
[the remark this answers is not in the document]
Comment (roy100) [reply docs/DTC Snippets.md]: done

File: docs/Data Transform Config (DTC).md [hunk 3]
```diff
@@ -31,7 +31,11 @@ When: source.package_version = '1.0'
 Key |  Description | Allowed values | Required
 --- | ------------ | -------------- | --------
 Type | The type of DTC - Streaming or Window | `Blurr:Streaming` or `Blurr:Window` | Required
+<<<<<<< HEAD:docs/Data Transform Config (DTC).md
```
[the remark this answers is not in the document]
Comment (roy100) [reply docs/Data Transform Config (DTC).md]: done

File: docs/DTC Snippets.md [hunk 4]
```diff
@@ -45,4 +45,18 @@ Fields:
     Value: item_stats.products_bought.extend(list(zip([val.strip() for val in source.item_id.split(',')], [float(val.strip()) for val in source.value.split(',')])))
 >>>>>>> master
```
Comment (mserranom) [inline docs/DTC Snippets.md]: this is a merge problem

**Pull request merged**